### ISSUE TITLE
German öäü not working in Google Doc / Sheets

### ISSUE THREAD

**Issue by MatelsoMan** [jpt/barlow#104]
Hi,

We love the font but are experiencing issues like the one mentioned here:
https://github.com/jpt/barlow/issues/47

Unfortunately, we’re using Google Workspace and can't update or modify anything within their system. I just wanted to check if we need to switch to a different font or if there might be any other solutions.

Apologies if this is outside of your scope, as it seems like Google may have just picked up your open license font. I'm not really well-versed in this, just an IT guy trying to make things work again.

But it’s a great font nonetheless!

**Comment by kenmcd** [comment]
This is probably from nested components (in TTF fonts).
Some printer drivers cannot handle nested components.
And you get accents placed in odd places - like the images in that other issue.

The `ä` is made of two components - `a` and the combining dieresis `̈` (`dieresiscomb`).
And the combining dieresis is made of two components - two combining dot accents `̇` (`dotaccentcom`).
So you have components inside other components.
Or nested components.

One work-around is to use the OTF fonts, because OTF fonts do not use components.
But since you are using Google Workspace, you are stuck with the Google Fonts (GF) fonts, which are all TTFs, and their current versions have this issue.
The bad news is until the font developer here updates the fonts, and removes the nested components - and then GF updates their fonts - you are kinda stuck.

Or use a different printer... have seen this with HP and Konica printers.
You could try updating the printer drivers.
But I think this issue is actually from some old code in the hardware, not the driver.

Note: nested components are allowed according to OpenType standards, but in the real world old embedded printer code cannot handle it.

**Comment by jpt** [comment]
yeah, it's both within and outside my scope. the good news is that Glyphs now has a custom parameter for removing nested components, which means this behavior either is (or will be) supported in fontmake, which Google now requires its fonts to be built with. so it will make its way in, it's just hard to say exactly when. I can add the custom parameter to the 1.5 branch to make sure it is in the next batch of changes that make their way to Google, at least

**Comment by MatelsoMan** [comment]
Thank you both this kind of verification of my findings exactly helps me to proceed from here just need to find out now how often google does updates and how we go from here maybe even really buy at least one new printer and be done with it. Thanks for adding it to 1.5!

**Comment by khaledhosny** [comment]
`fontmake` already has `--flatten-components` option.

**Comment by kenmcd** [comment]
> maybe even really buy at least one new printer and be done with it.

Be careful - test it first.
Some new printers still have very old code in the hardware.
My understanding is the on-the-fly conversion from TrueType to PostScript while printing is the root of the problem. And thus may be in really old hardware code.
Take your laptop and test the printer first.

**Comment by gstangl** [comment]
We have had the same problem in our company, and it drives us crazy. All with new Konica Minolta printers. We love Barlow and need Google Docs. The new Barlow 1.5 is working now. So when I export the Google Doc as Word file and print it, it shows all correctly. I hope Google updates the font soon.

**Comment by gstangl** [comment]
Is there a way to nudge the Google Workspace team to update Barlow to version 1.5?

**Comment by jpt** [comment]
It's more complicated than that, as I explained in your other issue [here](https://github.com/jpt/barlow/issues/96#issuecomment-2676211609). The first step is determining whether or not Barlow is even compatible with Google's font build requirements anymore. Even if it is, it will require some work to set up.
